docs/Configuring_openshift_virtualization_vm_for_dpdk.md

```diff
@@ -0,0 +1,281 @@
+...
+      domain:
+        resources:
+          requests:
+            memory: 8Gi
+        memory:
+          hugepages:
+            pageSize: "1Gi"
+...
+
+```
+Note: In this snippet we request 8 1Gi sized Hugepages.
+
+#### VM template summary example
+
+In the end the VM template should look like this:
+```yaml
+apiVersion: kubevirt.io/v1
+kind: VirtualMachine
+metadata:
+  name: rhel9-dpdk-vm
+spec:
+  dataVolumeTemplates:
+    - apiVersion: cdi.kubevirt.io/v1beta1
+      kind: DataVolume
+      metadata:
+        creationTimestamp: null
+        name: rhel9-dpdk-vm
+      spec:
+        sourceRef:
+          kind: DataSource
+          name: rhel9
+          namespace: openshift-virtualization-os-images
+        storage:
+          resources:
+            requests:
+              storage: 30Gi
+  running: true
+  template:
+    metadata:
+      annotations:
+        cpu-load-balancing.crio.io: disable
+        cpu-quota.crio.io: disable
+        irq-load-balancing.crio.io: disable
+    spec:
+      domain:
+        cpu:
+          sockets: 1
+          cores: 5
+          threads: 2
+          dedicatedCpuPlacement: true
+        devices:
+          disks:
+            - disk:
+                bus: virtio
+              name: rootdisk
+            - disk:
+                bus: virtio
+              name: cloudinitdisk
+          interfaces:
+            - masquerade: {}
+              name: default
+            - model: virtio
+              name: nic-east
+              pciAddress: '0000:07:00.0'
+              sriov: {}
+          networkInterfaceMultiqueue: true
+          rng: {}
+        memory:
+          hugepages:
+            pageSize: 1Gi
+        resources:
+          requests:
+            memory: 8Gi
+      networks:
+        - name: default
+          pod: {}
+        - multus:
+            networkName: dpdk-net
+          name: nic-east
+      terminationGracePeriodSeconds: 180
+      volumes:
+        - dataVolume:
+            name: rhel9-dpdk-vm
+          name: rootdisk
+        - cloudInitNoCloud:
+            userData: |-
+              #cloud-config
+              user: cloud-user
+              password: redhat
+              chpasswd: { expire: False }
+          name: cloudinitdisk
+```
+
+## Guest Configuration
+
+Once the VM is deployed and running, there are configurations that need to be set once on the guest. The changes are explained on each subchapter with appropriate snippets. In the end, an example guest configuration summary script is offered.
+
+### CPU isolation on guest
+
+If we continue with the example from previous chapters, 5 hyper-threads translate to 10 CPUs on the guest:
+```bash
+$ cat /sys/fs/cgroup/cpuset/cpuset.cpus
+0-9
+
+```
+If you have housekeeping processes running on the guest then at least 1 hyper thread should be left as non-isolated. Assuming this is true, the first CPU and its sibling should not be isolated:
+```bash
+$ cat /sys/devices/system/cpu/cpu0/topology/core_cpus_list
```

Comment:
Same.

Reply:
Could you please remove the `$` sign?
It is more convenient to copy the command using double-click when it is not there.

Reply:
DONE (missed one)